import_craft.py

```diff
@@ -0,0 +1,291 @@
+
+import os
+import bpy
+import import_mu
+from cfgnode import ConfigNode
+
+
+def print_blender_data_stats(label):
+    nbobj = len(bpy.data.objects)
+    nbmat = len(bpy.data.materials)
+    nbtex = len(bpy.data.textures)
+    nbimg = len(bpy.data.images)
+
+    print ('Loading stats ({}) :\
+            \n - {} objects \n - {} materials \
+            \n - {} textures \n - {} images \
+            \n'.format(label, nbobj, nbmat, nbtex, nbimg))
+
+
+def rename_textures(ob):
+    index = 0
+    for tex in bpy.data.textures:
```

Comment:
`ob` is used to rename the texture (l.25). Yes, renaming is made while data are retrieved from a .mu file, to avoid data merging when reading further .mu files. Using the name of the main object allows to have a specific name and also keep the link bewteen the blender data and the part they belong to. BTW, tell me if I misunderstood your comment.